api/src/test/resources/application.yaml

```diff
@@ -109,9 +109,9 @@ endpoint:
     all-commonschools:
       url: http://test
     school-by-min-code:
-      url: http://test
+      url: https://educ-grad-trax-api-77c02f-dev.apps.silver.devops.gov.bc.ca/api/v2/trax/school-clob/%s
```

Comment:
@infstar is there a reason to have the actual url here? I don't think unit tests should be calling another service and seems best to avoid having our api urls in the repo

Reply:
Yes, absolutely you are right. If you look at other urls in test yaml, it uses mainly for Dev.   It is handy to use the same url here to build the local yaml file against Dev quickly.